docs/linux/BUILDING.md

```diff
@@ -0,0 +1,46 @@
+# CentOS 7:
+
+CentOS 7 ships with older versions of glibc, cmake and gcc that make it difficult to compile this project and its dependencies. Building on CentOS involves providing newer packages built by the upstream projects, or though an available Software Collection Library(SCL).
+
+## Example: Building the `ziti-edge-tunnel` binary:
+
+
+```bash
+#We'll install CMake to this prefix, so let's make sure it's in the PATH
+export PATH="/opt/cmake:${PATH}"
+
+# Install project CMake required version
+curl -L https://cmake.org/files/v3.14/cmake-3.14.0-Linux-x86_64.sh -o ./cmake.sh
```

Comment:
are you getting 3.14 because it is the minimum required version for the tsdk project. If so I think it would be fine to get the latest CMake version that runs on Centos.

Reply:
I was indeed using this version for that reason. I've bumped it up to the version sued by the GitHub Actions Ubuntu 18.04 runner.